### PR TITLE
Feat: healthcheck

### PR DESCRIPTION
**Pull request by JMorinPS**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Please be specific when describing this change. What did you change? Why?
| Type?             | bug fix / improvement / new feature / refactor
| BC breaks?        | yes / no
| Deprecations?     | yes / no
| Fixed ticket?     | Fixes #{issue URL here}, Fixes #{another issue URL here}
| Sponsor company   | Your company or customer's name goes here (if applicable).
| How to test?      | Indicate how to verify that this change works as expected.

### CODE REVIEW

**Comment by JMorinPS** [comment]
Original code from @boherm on the following pull request : https://github.com/PrestaShop/kanbanbot/pull/16

File: config/services.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
 
 return function (ContainerConfigurator $configurator) {
     $configurator->parameters()
+        ->set('app.version', '1.1.0')
```
Comment (matks) [inline config/services.php:20]: Each time we deploy Kanbanbot, we publish a new Release on GitHub (this is how the CD works).

So we need to write somewhere a "How to deploy Kanbanbot" and insert there "DONT FORGET TO UPDATE THE VERSION NUMBER IN config/services.php"

Else we'll have a different version number on github and on the healthcheck 😅

File: src/Shared/Controller/HealthcheckController.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace App\Shared\Controller;
```
Comment (matks) [inline src/Shared/Controller/HealthcheckController.php:3]: Outside of the scope of this PR: we need to add the MIT license headers and the LICENSE.md

**Review by matks — COMMENTED** [review]
Can we have a test 😁 ? 

https://symfonycasts.com/screencast/symfony-bundle/controller-functional-test

**Comment by matks** [comment]
Woups reviewed the wrong PR

**Comment by boherm** [comment]
Closed in favor of https://github.com/PrestaShop/kanbanbot/pull/16.

**Pull request closed without merging**